packages/demand-bloc-payment/src/providers/AdyenProvider.ts

```diff
@@ -211,11 +211,7 @@ export class AdyenProvider implements Provider {
     return Promise.resolve('')
   }
 
-  async completeThreeDSecureVerification(params?: CompleteThreeDSecureVerificationParams) {
-    if (!params) {
-      return new Error(errors.missingRequiredParamsFor3dSecure)
-    }
-
```

Comment:
should not be deleted

Reply:
On there first glance it seems useless to count this param as optional and then check if it was provided but I think it might be used as some sort of indicator that params where not provided to the method from wherever it was called hmm